### ISSUE TITLE
Pretty logo

### ISSUE THREAD

**Issue by pkofod** [pkofod/MultiStart.jl#2]
I need a pretty logo, probably something like a contour plot with random points :)

**Comment by pkofod** [comment]
Maybe something like 
![Uploading couldbe.png…]()

**Comment by pkofod** [comment]
![couldbe](https://user-images.githubusercontent.com/8431156/96724154-cfb6cb80-13af-11eb-98e1-aa9fda67d4be.png)

**Comment by cormullion** [comment]
My first sketch. I got as far as browsing through the ColorSchemes.jl list to find a good colorscheme that works as a background, but perhaps you have more tasteful color preferences than me (I like all colors... 😃).

<img width="347" alt="Screenshot 2020-10-21 at 17 37 25" src="https://user-images.githubusercontent.com/52289/96750650-1adad980-13c4-11eb-981e-17d974abb8a9.png">

<img width="495" alt="Screenshot 2020-10-21 at 17 43 21" src="https://user-images.githubusercontent.com/52289/96751339-f59a9b00-13c4-11eb-9873-8e0f09d00e05.png">

**Comment by pkofod** [comment]
I really like the first one, maybe with opaque or white "background"?

**Comment by cormullion** [comment]
Yes, they were a bit dark... :) This is the `ColorSchemes.tableau_green_blue_white`

<img width="814" alt="Screenshot 2020-10-22 at 11 16 05" src="https://user-images.githubusercontent.com/52289/96858314-00ead680-1458-11eb-8db9-539f454ab416.png">
